examples/multi-cassdc-single-k8s/README.md

```diff
@@ -0,0 +1,69 @@
+# Example: Multiple Cassandra datacenters in a single Kubernetes cluster
+
+This example configuration deploys a Cassandra cluster consisting of two Cassandra datacenters with three nodes per datacenter. Each datacenter is deployed in a separate namespace within a shared Kubernetes cluster. 
+
+## Background and setup
+This example is designed to run on a Google Kubernetes Engine (GKE) cluster in the `us-west4` region. Please see the [Google GKE](https://docs.k8ssandra.io/install/gke/) page for instructions on setting up an appropriate cluster you can use with this example, tailoring the zone/region settings as needed. 
+
+Alternatively, you can make this example more generic and run on another Kubernetes engine by removing the affinity lines in the yaml files.
+
+These instructions assume you've installed Helm and that your Kubernetes context is set to the cluster you want to use. Consult the [Getting Started](https://k8ssandra.io/get-started) documentation for more information.
+
+The instructions guide you through creating two Cassandra datacenters in a single Kubernetes cluster, one designated for transactional workloads (`txndc`), and one for analytics workloads (`analyticsdc`).
+
+## Instructions
+
+1. Create the namespaces and administrator credentials for each datacenter:
+
+    ```
+    kubectl create namespace txndc
+    kubectl create secret generic cassandra-admin-secret --from-literal=username=cassandra-admin --from-literal=password=cassandra-admin-password -n txndc
+    kubectl create namespace analyticsdc
+    kubectl create secret generic cassandra-admin-secret --from-literal=username=cassandra-admin --from-literal=password=cassandra-admin-password -n analyticsdc
+    ```
+
+1. Install K8ssandra in the transactional namespace for the second datacenter using the [dc1.yaml](./dc1.yaml) config file:
+
+    ```
+    helm install txndc k8ssandra/k8ssandra -f dc1.yaml -n txndc  
+    ```
+
+1. After the first datacenter has initialized (you can watch status using `watch kubectl get pods -n txndc`), install the second datacenter in the analytics namespace using the [dc2.yaml](./dc2.yaml) config file. 
```

Comment:
On the watch status, you could either replace or offer-up another option (which supports `winos`) used directly w/ kubectl.

```
kubectl get pods -w ...
```

_Description_:
-w, --watch=false: After listing/getting the requested object, watch for changes. Uninitialized objects are excluded
if no object name is provided.